python/F3DPythonBindings.cxx

```diff
@@ -268,9 +268,12 @@ PYBIND11_MODULE(pyf3d, module)
     .def("pan", &f3d::camera::pan, py::arg("right"), py::arg("up"), py::arg("forward") = 0.0)
     .def("zoom", &f3d::camera::zoom)
     .def("roll", &f3d::camera::roll)
-    .def("azimuth", &f3d::camera::azimuth)
-    .def("yaw", &f3d::camera::yaw)
-    .def("elevation", &f3d::camera::elevation)
+    .def("add_azimuth", &f3d::camera::addAzimuth)
+    .def("add_yaw", &f3d::camera::addYaw)
+    .def("add_elevation", &f3d::camera::addElevation)
+    .def("get_azimuth", &f3d::camera::getAzimuth)
+    .def("get_yaw", &f3d::camera::getYaw)
+    .def("get_elevation", &f3d::camera::getElevation)
```

Comment:
that could be properties, see `image` class for examples
```suggestion
    .def_property_readonly("azimuth", &f3d::camera::getAzimuth)
    .def_property_readonly("yaw", &f3d::camera::getYaw)
    .def_property_readonly("elevation", &f3d::camera::getElevation)
```

Reply:
Implemented.